src/utils/email.js

```diff
@@ -0,0 +1,90 @@
+//@ts-check
+import * as ejs from 'ejs'
+import * as fs from 'fs'
+import * as path from 'path'
+import { createTransport } from 'nodemailer'
+import { minify } from 'html-minifier'
+const { google } = require('googleapis')
+
```

Comment:
Add here to solve the error i'm facing
```suggestion
require('dotenv').config()
```

Reply:
Okay I will